test/taoensso/faraday/tests/requests.clj

```diff
@@ -219,6 +233,13 @@
     (expect "global-secondary" (.getIndexName action))
     ))
 
+(let [req ^UpdateTableRequest (update-table-request
```

Comment:
Type hints only work when attached to the symbol, otherwise they're ignored. If other tests have the incorrect form then the type hints are either unnecessary or need to be fixed; will check in a moment. Thanks for the report!